nharsch_changes.md

```diff
@@ -0,0 +1,22 @@
+Here's what I tried:
```

Comment:
@ericbey I made a couple changes to a couple of the files you called out, here are my notes